tool/travis_flutter_script.sh

```diff
@@ -59,10 +59,13 @@ do
 done
 
 # Test that the code segment widgets that get displayed in the Flutter Material
-# gallery have been generated using the latest gallery code.
+# gallery have been generated using the latest gallery code. It also tests that
+# the localization scripts have been run, so that they are up to date for the
+# gallery.
 echo "Run code segments check for 'gallery/gallery'."
```

Comment:
Maybe you should add an additional `echo` to indicate the l10n script is running after the code segments one?

Reply:
The grinder tool will output:

```
grinder running verify-l10n

verify-l10n

finished in 0.2 seconds
```

So I am not sure that it is necessary, see [log here](https://travis-ci.org/flutter/samples/jobs/642329914?utm_medium=notification&utm_source=github_status).

Reply:
Oh wait. Actually I agree, since we have the echo for code segments. Let me add this.